apps/prairielearn/src/server.js

```diff
@@ -1869,19 +1869,16 @@ export async function initExpress() {
     (await import('./pages/instructorGradingJob/instructorGradingJob.js')).default,
   );
 
-  // This route is used to initiate a transfer of a question from a template course.
+  // This route is used to initiate a copy of a question with publicly shared source
+  // or a question from a template course.
   // It is not actually a page; it's just used to initiate the transfer. The reason
   // that this is a route on the target course and not handled by the source question
   // pages is that the source question pages are served by chunk servers, but the
   // question transfer machinery relies on access to course repositories on disk,
   // which don't exist on chunk servers
   app.use(
-    '/pl/course/:course_id(\\d+)/copy_template_course_question',
-    (
-      await import(
-        './pages/instructorCopyTemplateCourseQuestion/instructorCopyTemplateCourseQuestion.js'
-      )
-    ).default,
```

Comment:
Removing a route like this is generally not safe; what happens when someone tries to copy a question during a deploy? I agree that we want to rename this route, but we should do it with roughly the same strategy we'd use for any database columns:

- Add the new route; deploy
- Update anything referencing the old route to instead point to the new route; deploy
- Remove the old route

Reply:
done here:

https://github.com/PrairieLearn/PrairieLearn/pull/9918
https://github.com/PrairieLearn/PrairieLearn/pull/9919